### PR TITLE
feat(operations): persist operation audit details

### PR DESCRIPTION
**Pull request by dimileeh**
Automatically opened by AWF workspace `ws_83f4e614951446cf883f5c09` (agent: `codex`, model: `gpt-5.5`, effort: `xhigh`).

**External task ID**: pre-gke-p0-operation-audit-codex-20260428130653

### Task
Implement the next P0 Operation And Recovery Truth slice from TODO/pre-gke-industrial-readiness.md using strict TDD.

Goal:
Persist richer operation audit truth for refresh, rebase, validate, remonitor, cancel/stop/delete, and recovery operations, so operators can see start, finish, owner/source, reason, result, failure code, and log stream references without scraping raw logs.

Required behavior:
- Extend existing Operation records/repositories/services to preserve structured audit details for operation owner/source, reason code, result payload, failure code/message, started_at/finished_at where not already present, and associated log stream ids where available.
- Do not introduce a new table unless needed; prefer compatible columns/payload structure if existing schema supports it. If a migration is necessary, add it with tests.
- Ensure operation list/detail API returns these fields in a stable response shape while preserving backwards compatibility.
- Make recovery/monitor-created operations include source=pr_monitor or equivalent owner and actionable reason.
- Add tests for successful operation completion, failed operation completion, idempotent duplicate operation preserving original audit fields, and API serialization.
- Keep the slice scoped to operation audit truth; do not implement console controls here.

TDD requirements:
- Write failing tests first.
- Use focused fake sessions/services where possible.
- Do not lower coverage thresholds, .awf coverage settings, pyproject fail_under, or quality gates.
- Do not push; AWF owns push, PR creation, monitoring, comments, and merge.

---
Validation: 7 profile command(s) passed inside the workspace container.

### CODE REVIEW

File: src/awf/api/schemas.py [hunk 1]
```diff
@@ -690,6 +690,78 @@ class OperationResponse(BaseModel):
...
+
+
+def _operation_log_stream_refs(
+    *sources: dict[str, Any],
+) -> dict[str, Any]:
+    refs: dict[str, Any] = {}
+    for source in sources:
+        value = source.get("log_stream_refs")
+        if isinstance(value, dict):
+            refs.update(value)
```
[the remark this answers is not in the document]
Comment (dimileeh) [reply src/awf/api/schemas.py]: fixed in commit 335d0256197f02ffee8de71b361fb973a1dffc13

File: src/awf/api/schemas.py [hunk 2]
```diff
@@ -690,6 +690,78 @@ class OperationResponse(BaseModel):
...
+            return
+        if isinstance(item, list | tuple):
+            for child in item:
+                collect(child)
+
+    collect(value)
+    return sorted(ids)
 
 
 class OperationListResponse(BaseModel):
```
[the remark this answers is not in the document]
Comment (dimileeh) [reply src/awf/api/schemas.py:800]: fixed in commit 62f8c76ba13a6110225e43c1cd8ae702f8a72d3f

File: src/awf/runtime/pr_monitor_runner.py [hunk 3]
```diff
@@ -1221,11 +1221,17 @@ async def _handle_merge_gate_blocker(
 
         if stale_reason is not None:
             recovery_mode = "rebase_only" if req_action == "rebase" else "validate_only"
+            requested_action = req_action or "validate"
             operation_payload: dict[str, object] = {
+                "owner": "pr_monitor",
                 "source": "pr_monitor",
                 "reason": stale_reason,
+                "reason_code": stale_reason,
+                "requested_action": requested_action,
```
[the remark this answers is not in the document]
Comment (dimileeh) [reply src/awf/runtime/pr_monitor_runner.py:1233]: fixed in commit 1e3362f085bab303d627784282248fbd6c0e467e
